src/main/scala/uk/gov/hmrc/HeaderUtils.scala

```diff
@@ -52,8 +55,11 @@ object HeaderUtils {
         FileUtils.readFileAsString(licenceFile).toEither.left.map(_ => s"Problem reading LICENSE file")
           // .right projection required to remain backwards compatible with scala 2.10 cross build (for sbt 0.13)
           .right.flatMap(c =>
-          if (Seq(licenceFileExpectedMD5, licenceFileExpectedMD5_2).contains(md5HashString(c.trim))) Right(())
-          else Left(s"The LICENSE file does not contain the appropriate Apache V2 licence. It should match https://www.apache.org/licenses/LICENSE-2.0.txt")
+          Either.cond(
+            Seq(licenceFileExpectedMD5, licenceFileExpectedMD5_Windows, licenceFileExpectedMD5_2, licenceFileExpectedMD5_2_Windows).contains(md5HashString(c.trim)),
+            right = (),
+            left = s"The LICENSE file does not contain the appropriate Apache V2 licence. It should match https://www.apache.org/licenses/LICENSE-2.0.txt"
+          )
```

Comment:
@christopherjturner re: [this](https://hmrcdigital.slack.com/archives/G0HRAFPN3/p1630587273003000?thread_ts=1630586900.002500&cid=G0HRAFPN3), how does this sound?  

```scala
s"The LICENSE file does not contain the appropriate Apache V2 licence as the computed md5HashString: ${c.trim} does not match that of: https://www.apache.org/licenses/LICENSE-2.0.txt
```